specs/commandline/plugin.md

```diff
@@ -41,51 +42,87 @@ Aliases:
 ### notation plugin install
 
 ```text
-Installs a plugin
+Install a plugin
 
 Usage:
-  notation plugin install [flags] <plugin path>
+  notation plugin install [flags] <plugin location>
 
 Flags:
   -h, --help                    help for install
   -f, --force                   force the installation of a plugin
-
+      --checksum                verify the checksum digest of a plugin
+      --source string           the location of plugin installation file, options: "file", "url","registry" (default "file")                  
```

Comment:
To align with other designs, can we use separate flags? Something like:
```suggestion
      --file bool           if set, install the plugin from file system
      --url bool            if set, install the plugin from URL
```
We could simply make these flags mutual exclusive. (Based on PRD, we will support two flags in the first iteration.)

Reply:
Having a `--source` flag with different options"file", "url","registry" (default "file") seems simpler than providing multiple flags here.